### PR TITLE
Review send key calls

### PR DESCRIPTION
**Pull request by coolo**

### CODE REVIEW

File: tests/x11/thunderbird.pm [hunk 1]
```diff
@@ -16,8 +16,10 @@ sub run() {
     ensure_installed("MozillaThunderbird");
     x11_start_program("thunderbird");
     assert_screen 'test-thunderbird-1', 3;
-    send_key "alt-f4",                  1;    # close wizzard
-    send_key "alt-f4",                  1;    # close prog
+    wait_screen_change {
+        send_key "alt-f4";    # close wizzard
+    }
+    send_key "alt-f4";        # close prog
```
Comment (okurz) [inline tests/x11/thunderbird.pm]: * shouldn't both be guarded with wait_screen_change?
* Didn't you tell me that "wait_screen_change {}" needs to be closed with ';'?

File: tests/x11regressions/gedit/gedit_about.pm [hunk 2]
```diff
@@ -18,7 +18,9 @@ sub run() {
     x11_start_program("gedit");
 
     # check about window
-    send_key "alt-h", 1;
+    wait_screen_change {
+        send_key "alt-h";
+    }
```
Comment (okurz) [inline tests/x11regressions/gedit/gedit_about.pm]: ;

File: tests/x11regressions/gedit/gedit_about.pm [hunk 3]
```diff
@@ -29,14 +31,18 @@ sub run() {
     assert_and_click 'gedit-about-link';
     # give a little time to open and load website
     assert_screen 'gedit-open-firefox', 60;
-    send_key "ctrl-q",                  1;
+    wait_screen_change {
+        send_key "ctrl-q";
+    }
```
Comment (okurz) [inline tests/x11regressions/gedit/gedit_about.pm]: ;

File: tests/x11/thunderbird.pm (hunk 1 above)
Comment (coolo) [inline tests/x11/thunderbird.pm]: the end of the test is guarded by an assert_screen 'generic-desktop' from the base class. And the missing ; was done before I noticed the problem - but I left it in to check your reviews ;)

Thx

**Comment by coolo** [comment]
added the ;s

File: tests/x11regressions/gedit/gedit_save.pm [hunk 4]
```diff
@@ -43,13 +43,17 @@ sub run() {
...
+    };
     send_key "ctrl-q";
 
     # open saved file to validate
     x11_start_program("gedit " . "test.txt");
     assert_screen 'gedit-saved-file', 3;
-    send_key "ctrl-q",                1;
+    wait_screen_change {
+        send_key "ctrl-q";
+    }
```
Comment (okurz) [inline tests/x11regressions/gedit/gedit_save.pm]: ;

maybe it looks safer it we write this special construct in one line
```
wait_screen_change { send_key "ctrl-q"; };
```
or even
```
wait_screen_change { send_key "ctrl-q" };
```

Comment (MalloZup) [inline tests/x11regressions/gedit/gedit_save.pm]: +1

**Comment by coolo** [comment]
changed to one liners in gedit_save

**Comment by okurz** [comment]
LGTM

travis failed to download some stuff, retry, should work.

**Comment by coolo** [comment]
no risk, no fun

**Pull request merged**